src/modules/fundingManager/bondingCurve/FM_BC_BondingSurface_Redeeming_v1.sol

```diff
@@ -44,312 +43,394 @@ import {SafeERC20} from "@oz/token/ERC20/utils/SafeERC20.sol";
+        external
+        virtual
+        override(IFundingManager_v1)
+        onlyPaymentClient
+    {
+        if (
+            amount_
+                > token().balanceOf(address(this)) - projectCollateralFeeCollected
+        ) {
+            revert InvalidOrchestratorTokenWithdrawAmount();
+        }
+
+        token().safeTransfer(to_, amount_);
+
+        if (MIN_RESERVE > token().balanceOf(address(this))) {
+            revert FM_BC_BondingSurface_Redeeming_v1__MinReserveReached();
+        }
+
+        emit TransferOrchestratorToken(to_, amount_);
+    }
+
+    // ------------------------------------------------------------------------
+    // Mutating - Upstream Function Implementations
+
+    /// @notice Calculates the amount of tokens to mint for a given deposit
+    ///         amount using the formula contract.
+    /// @dev This internal function is an override of BondingCurveBase_v1's
+    ///      abstract function.
+    /// @param depositAmount_   The amount of collateral deposited to
+    ///                         purchase tokens.
+    /// @return mintAmount_ The amount of tokens that will be minted.
+    function _issueTokensFormulaWrapper(uint depositAmount_)
         internal
         view
         override(BondingCurveBase_v1)
-        returns (uint mintAmount)
+        returns (uint mintAmount_)
     {
         uint capitalAvailable = _getCapitalAvailable();
         if (capitalAvailable == 0) {
             revert FM_BC_BondingSurface_Redeeming_v1__NoCapitalAvailable();
         }
 
-        mintAmount = formula.tokenOut(
-            _depositAmount, capitalAvailable, basePriceToCapitalRatio
+        mintAmount_ = _formula.tokenOut(
+            depositAmount_, capitalAvailable, _basePriceToCapitalRatio
         );
     }
 
-    /// @dev Calculates the amount of collateral to be received when redeeming a given amount of tokens.
-    /// This internal function is an override of RedeemingBondingCurveBase_v1's abstract function.
-    /// @param _depositAmount The amount of tokens to be redeemed for collateral.
-    /// @return redeemAmount The amount of collateral that will be received.
-    function _redeemTokensFormulaWrapper(uint _depositAmount)
+    /// @notice Calculates the amount of collateral to be received when
+    ///         redeeming a given amount of tokens.
+    /// @dev    This internal function is an override of
+    ///         RedeemingBondingCurveBase_v1's abstract function.
+    /// @param depositAmount_   The amount of tokens to be redeemed for
+    ///                         collateral.
+    /// @return redeemAmount_   The amount of collateral that will be received.
+    function _redeemTokensFormulaWrapper(uint depositAmount_)
         internal
         view
         override(RedeemingBondingCurveBase_v1)
-        returns (uint redeemAmount)
+        returns (uint redeemAmount_)
     {
         // Subtract fee collected from capital held by contract
```

Comment:
```suggestion
        // Subtract fee collected from capital held by contract.
```

Reply:
Btw, am I misunderstanding something or how does this mention fees but there is nothing happening with fees here?